data/remnant/remnant missions.txt

```diff
@@ -3463,3 +3463,279 @@ event "remnant: pantica void sprites"
+			choice
+				`	"I don't know where that is."`
+					goto center
+				`	"Isn't that the center of the galaxy?"`
+					goto centeralt
+				`	"I have been there."`
+			`	"Really?" She looks surprised. "I have seen sensor logs and recordings from previous visits, but not many people have actually been there." She pauses to take a bite from her neglected meal. "Being able to take part in these fieldwork expeditions is something I have been looking forward to for ages, so of course Sagittarius A* is the first place on my list."`
+				goto center
+			label centeralt
+			`	"Yes, that is correct." She signs firmly.`
+			label center
+			`	Her signs take on a rhythmic pace as if reciting something. "Sagittarius A* is a strong astronomical radio source and a supermassive black hole that is located at the center of the Milky Way. According to our best measurements, it has a mass of 3F6290 solar masses and is surrounded by a ring of gas and dust that can range from 63 to 24D2 degrees Kelvin."`
+			`	She returns to a conversational cadence. "Would you be interested in taking part in this expedition?"`
+			choice
+				`	"Sure, what do you need?"`
+					goto scienceequipment
+				`	"You want me to take you to a black hole?!"`
+				`	"Sorry, I'm not interested in visiting a black hole."`
+					decline
+			`	She looks momentarily surprised by your reaction. "Yes, exactly that. Current gravitational repulsor engines can nullify most of the gravitational pull, and an extra thermoelectric cooler or two should be sufficient to manage the heat. As such, the risks are mangeable so long as you do not try to approach the event horizon.`
+			label scienceequipment
+			`	"For this mission in specific, however, we will need some scientific equipment that is actually hardwired into your ship, not just carried along with us. When we arrive on <planet>, they will have the equipment ready to install. Just be aware that it will require six outfit space, and requires a scientist, so you will need a free bunk for them, too.`
+			`	"Once the laboratory module is installed, it will simply be a matter of traveling to Sagittarius A*. After our measurements, recordings, and observations are complete, we will return and report on our findings."`
+			`	She glances down at her half-finished meal. "I am already packed and ready to go. I will meet you onboard the <ship> as soon as I am done eating."`
+			branch dawnsymbiont
+				has "remnant symbionts info"
+			choice
+				`	"Okay, I will meet you onboard the ship."`
+					accept
+				`	"Developed quite the appetite, haven't you?"`
+			branch dawnlicense
+				has "license: Remnant Capital"
+			`	She shrugs. "I burn a lot of calories, and eating healthy is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
```

Comment:
```suggestion
			`	She shrugs. "I burn a lot of calories, and healthy eating is important. Besides, I don't know how good food will be on your ship." She resumes eating with vigor and does not seem inclined to continue the conversation.`
```

("eating healthy" is colloquial but grammatically incorrect, and her speech patterns indicate a preference for precision)